.github/workflows/rust.yml

```diff
@@ -82,6 +82,8 @@ jobs:
         uses: ./.github/actions/setup-builder
         with:
           rust-version: stable
+      - name: Check datafusion-common (default features)
```

Comment:
there was no check for default set of features (even the tests run with avro, etc) so I added that

Reply:
good finding!